src/utils/components/SSHSecretSection/utils/utils.ts

```diff
@@ -169,3 +193,49 @@ export const getPropagationMethod = (
   vm: V1VirtualMachine,
 ): V1SSHPublicKeyAccessCredentialPropagationMethod =>
   vm?.spec?.template?.spec?.accessCredentials?.[0].sshPublicKey.propagationMethod;
+
+const generateShortPrettyName = (prefix?: string): string => {
+  const numberDictionary = NumberDictionary.generate({ length: 2 });
+  const prefixValue = prefix ? `${prefix}-` : '';
+
+  return `${prefixValue}${uniqueNamesGenerator({
+    dictionaries: [animals, numberDictionary],
+    separator: '-',
+  })}`;
+};
```

Comment:
why duplicate it with a minor change?

Reply:
This was part of trying to keep as much of the original name as possible.